Slack.Automation/Promact.Core.Repository/ExternalLoginRepository/OAuthLoginRepository.cs

```diff
@@ -78,27 +84,29 @@ public OAuthApplication ExternalLoginInformation(string refreshToken)
             var slackUsers = JsonConvert.DeserializeObject<SlackUserResponse>(userDetailsResponse);
             foreach (var user in slackUsers.Members)
             {
-                if (user.Name != StringConstant.SlackBotStringName)
-                {
-                    user.CreatedOn = DateTime.UtcNow;
-                    _slackUserDetails.Insert(user);
-                    _slackUserDetails.Save();
-                }
+                if (!user.Deleted && !user.IsBot && user.Name != StringConstant.SlackBotStringName)
```

Comment:
Sir, otherwise slackbot would be added to SlackUserDetails tables, as it is neither marked(is_bot=false) as a bot by slack nor deleted,the two conditions that we check to be false while adding slack users to database.